android/src/main/java/com/paywallsdkreactnative/HeliumBridge.kt

```
@@ -0,0 +1,760 @@
+
+                Helium.initialize(
+                    context = reactContext,
+                    apiKey = apiKey,
+                    heliumPaywallDelegate = delegate,
+                    customUserId = customUserId,
+                    customApiEndpoint = customAPIEndpoint,
+                    customUserTraits = customUserTraits,
+                    fallbackConfig = fallbackConfig,
+                    environment = environment
+                )
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)